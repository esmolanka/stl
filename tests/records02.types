
type Record =
  { afield     : Unit
  , 'type'     : Unit
  , 'module'   : Unit
  , 'with'     : Unit
  , 'provides' : Unit
  }

#eval Record
