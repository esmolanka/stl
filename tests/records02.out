tests/records02.stl:10:1-10:13:
  Normalised:
    ∃ (ρ : Row).
      { afield : Unit
      , type : Unit
      , module : Unit
      , with : Unit
      , provides : Unit
      | ρ
      }
  Kind:
    Type